README.md

```diff
@@ -1,2 +1,34 @@
 # tfc-dump
-Perl command using the Terraform Cloud API to export workspaces, variables, and variable sets to JSON files.
+Perl script using the Terraform Cloud API to export workspaces, variables, and variable sets to JSON files.
+
+## Motivation
+During the review of a disaster recovery plan, we realized that we didn't have a
+record of the values we set for variables in Terraform Cloud workspaces.
+It would be difficult to recover from the accidental deletion of a Terraform
+Cloud workspace.
+This Perl script was quickly written to export information to JSON files
+using the Terraform Cloud API.
+The exported information includes workspaces, variables, and variable sets
+which covers most of what we need.
+
+## How to use tfc-dump.pl
+1. Install [tfc-ops](https://github.com/silinternational/tfc-ops).
+2. Obtain a Terraform Cloud access token.
+3. export ATLAS\_TOKEN=_terraform-cloud-access-token_
```

Comment:
This adds the token to shell history, which seems like a bad idea to me. Typically you can add a space before the command to avoid this, but that doesn't present well here. Maybe better to just include a reminder caution here.

Reply:
Good point, I added text and pointed the user to the bash man page.  Someday I'll need to finish reading the bash man page.